source/MaterialXRender/HardwarePlatform.h

```diff
@@ -0,0 +1,21 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_HARDWAREPLATFORM_H
+//
+// Platform macros. All platform specific files in the module should use the OS*_ macros
+// instead of the native ones.
+//
+#if defined(_WIN64) || defined(_WIN32)
```

Comment:
I'm wondering if we're really adding clarity with the definition of OSWin_, OSLinux_, and OSMac tags across MaterialXRender.  On Windows, it should always be sufficient to check for \_WIN32 (the "32" is just a legacy suffix, and this tag is defined in all Windows builds), while the \_\_linux\_\_ and \_\_APPLE\_\_ definitions are clear and sufficient on their respective platforms.

Perhaps our code would be clearer without the addition of HardwarePlatform.h at all, and we could use the three native preprocessor definitions wherever per-platform distinctions are needed?